File: financemanager.web/interfaces/api/pots/GetAddMonthPotDataDto.ts
export interface GetAddMonthPotDataDto {
  spendingPots: AddMonthSpendingPot[]
  savingsPots: AddMonthSavingsPot[]
}

export interface AddMonthSpendingPot {
  potId: number
  potName: string
  amountToAdd: number
  rolloverAmount: string
}

export interface AddMonthSavingsPot {
  potId: number
  potName: string
  amountToAdd: number
}

File: Orbit.Domain/DTOs/Pots/Responses/GetAddMonthPotDataDto.cs
namespace Orbit.Domain.DTOs.Pots.Responses
{
    public class GetAddMonthPotDataDto
    {
        public required AddMonthSpendingPot[] SpendingPots { get; set; }
        public required AddMonthSavingsPot[] SavingsPots { get; set; }
    }

    public class AddMonthSpendingPot
    {
        public required int PotId { get; set; }
        public required string PotName { get; set; }
        public required decimal AmountToAdd { get; set; }
        public required string RolloverAmount { get; set; }
    }

    public class AddMonthSavingsPot
    {
        public required int PotId { get; set; }
        public required string PotName { get; set; }
        public required decimal AmountToAdd { get; set; }
    }
}
